File: Python For Everyone, Horstmann/Chapter2-Programming-with-Numbers-and-Strings/P2.12.py
# File names and extensions. Write a program that prompts the user for the drive letter
# ( C ), the path ( \Windows\System ), the file name ( Readme ), and the extension ( txt ). Then
# print the complete file name  C:\Windows\System\Readme.txt . (If you use UNIX or a
# Macintosh, skip the drive name and use  / instead of \ to separate directories.)

driveLetter = str(input("Enter the drive letter: "))
path = str(input("Enter the path(folders only: "))
fileName = str(input("Enter the file name: "))
fileExtension = str(input("Enter the file extension: "))

print("The complete file name path is:")
print(driveLetter + ":\\" + path + "\\" + fileName + "." + fileExtension)

File: Java-for-Everyone-Late-Objects-Horstmann/Chapter-2-Fundamental-Data-Types/P2_12.java
/*File names and extensions. Write a program that prompts the user for the drive letter
( C ), the path ( \Windows\System ), the file name ( Readme ), and the extension ( txt ). 
Then print the complete file name  C:\Windows\System\Readme.txt . (If you use UNIX or a
Macintosh, skip the drive name and use  / instead of \ to separate directories.)
*/


import java.util.Scanner;


public class P2_12 {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        System.out.print("Please enter the drive letter: ");
        String driveLetter = in.next();
        System.out.print("Please enter the path: ");
        String path = in.next();
        System.out.print("Please enter the filename: ");
        String filename = in.next();
        System.out.print("Please enter the extension: ");
        String extension = in.next();
        in.close();
        String completeFilename = driveLetter + ":" + path + "\\" + filename + "." + extension;
        System.out.println(completeFilename);
    }
}
